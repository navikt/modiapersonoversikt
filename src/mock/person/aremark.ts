import { Diskresjonskoder, TilrettelagtKommunikasjonsTyper } from '../../konstanter';
import { BostatusTyper, Kjønn, Person, Relasjonstype, SivilstandTyper } from '../../models/person/person';
import { SikkerhetstiltakTyper } from '../../models/sikkerhetstiltak';
import { Personadresse } from '../../models/personadresse';

export const gateAdresseAremark: Personadresse = {
    endringsinfo: {
        sistEndret: '2017-11-25T21:29:14.049Z',
        sistEndretAv: 'SKD'
    },
    gateadresse: {
        gatenavn: 'Islandsgate',
        husnummer: '49',
        postnummer: '7693',
        poststed: 'Hovedøya'
    }
};

export const postboksAdresseAremark: Personadresse = {
    endringsinfo: {
        sistEndret: '2018-04-10T09:06:58.256Z',
        sistEndretAv: 'SKD'
    },
    postboksadresse: {
        postboksnummer: '9698',
        postnummer: '7705',
        poststed: 'Færder fyr'
    }
};

export const matrikkelAdresseAremark: Personadresse = {
    endringsinfo: {
        sistEndret: '2018-04-10T09:06:58.256Z',
        sistEndretAv: 'SKD'
    },
    matrikkeladresse: {
        postnummer: '5468',
        poststed: 'Lillomarka'
    }
};

export const utlandsAdresseAremark: Personadresse = {
    endringsinfo: {
        sistEndret: '2018-04-10T09:06:58.256Z',
        sistEndretAv: 'SKD'
    },
    utlandsadresse: {
        landkode: {
            kodeRef: 'SYD',
            beskrivelse: 'Syden'
        },
        adresselinjer: ['Ut av landet', 'Rundt svingen']
    }
};

export const aremark: Person = {
    fødselsnummer: '10108000398',
    kjønn: Kjønn.Mann,
    geografiskTilknytning: '0118',
    alder: 42,
    folkeregistrertAdresse: gateAdresseAremark,
    alternativAdresse: postboksAdresseAremark,
    bankkonto: {
        banknavn: 'Bien sparebank',
        kontonummer: '12345678911',
        sistEndret: '2005-12-12',
        sistEndretAv: 'z9910',
        landkode: {
            kodeRef: 'NOR',
            beskrivelse: 'Norge'
        },
        valuta: {
            beskrivelse: 'Spesidaler',
            kodeRef: 'SPD'
        }
    },
    sikkerhetstiltak: {
        sikkerhetstiltaksbeskrivelse:
            'Får verken møte opp på NAV-kontor eller ringe på telefon ' + 'grunnet truende oppførsel og språk.',
        sikkerhetstiltakskode: SikkerhetstiltakTyper.FysiskOgTelefonUtestengelse,
        periode: {
            fra: '2005-12-12',
            til: '2025-12-12'
        }
    },
    navn: {
        endringsinfo: {
            sistEndret: '2012-12-12',
            sistEndretAv: 'NAV'
        },
        sammensatt: 'AREMARK TESTFAMILIEN',
        fornavn: 'AREMARK',
        mellomnavn: '',
        etternavn: 'TESTFAMILIEN'
    },
    tilrettelagtKomunikasjonsListe: [
        {
            kodeRef: 'TOHJ',
            beskrivelse: TilrettelagtKommunikasjonsTyper.TOHJ
        }
    ],
    diskresjonskode: { kodeRef: Diskresjonskoder.FORTROLIG_ADRESSE, beskrivelse: 'Sperret adresse, fortrolig' },
    statsborgerskap: { kodeRef: 'NOR', beskrivelse: 'NORGE' },
    personstatus: {
        dødsdato: undefined,
        bostatus: {
            kodeRef: BostatusTyper.Utvandret,
            beskrivelse: BostatusTyper.Utvandret
        }
    },
    sivilstand: {
        kodeRef: SivilstandTyper.Gift,
        beskrivelse: 'Gift',
        fraOgMed: '2005-12-12'
    },
    familierelasjoner: [
        {
            rolle: Relasjonstype.Barn,
            tilPerson: {
                navn: null,
                diskresjonskode: {
                    kodeRef: Diskresjonskoder.FORTROLIG_ADRESSE,
                    beskrivelse: 'Sperret adresse, fortrolig'
                },
                personstatus: {}
            }
        },
        {
            rolle: Relasjonstype.Barn,
            tilPerson: {
                navn: null,
                diskresjonskode: {
                    kodeRef: Diskresjonskoder.STRENGT_FORTROLIG_ADRESSE,
                    beskrivelse: 'Sperret adresse, strengt fortrolig'
                },
                personstatus: {}
            }
        },
        {
            harSammeBosted: true,
            rolle: Relasjonstype.Barn,
            tilPerson: {
                fødselsnummer: '10117100000',
                navn: {
                    sammensatt: 'Siri Bellomnavn Aremark',
                    fornavn: 'Siri',
                    mellomnavn: 'Bellomnavn',
                    etternavn: 'Aremark'
                },
                personstatus: {},
                alder: 15
            }
        },
        {
            harSammeBosted: false,
            rolle: Relasjonstype.Barn,
            tilPerson: {
                fødselsnummer: '10117100100',
                navn: {
                    sammensatt: 'Ola Ballomnavn Aremark',
                    fornavn: 'Ola',
                    mellomnavn: 'Ballomnavn',
                    etternavn: 'Aremark'
                },
                personstatus: {},
                alderMåneder: 5
            }
        },
        {
            harSammeBosted: false,
            rolle: Relasjonstype.Ektefelle,
            tilPerson: {
                navn: null,
                diskresjonskode: {
                    kodeRef: Diskresjonskoder.STRENGT_FORTROLIG_ADRESSE,
                    beskrivelse: 'Sperret adresse, strengt fortrolig'
                },
                personstatus: {}
            }
        }
    ],
    kontaktinformasjon: {
        mobil: {
            sistEndret: '2014-06-21T18:44:39+02:00',
            identifikator: '99887766',
            retningsnummer: { kodeRef: '+47', beskrivelse: 'Norge' },
            sistEndretAv: '10108000398'
        }
    }
};
