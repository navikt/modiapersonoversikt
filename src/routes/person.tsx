import { Navigate, createFileRoute } from '@tanstack/react-router';
import LegacyAppContainer from 'src/app/LegacyAppContainer';
import PersonoversiktWrapper from 'src/app/personside/Personoversikt';

export const Route = createFileRoute('/person')({
    component: PersonLayout,
    notFoundComponent: () => <Navigate to="/person/oversikt" replace />
});

function PersonLayout() {
    return (
        <LegacyAppContainer>
            <PersonoversiktWrapper />
        </LegacyAppContainer>
    );
}
