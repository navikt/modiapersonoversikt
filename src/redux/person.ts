import { Person } from '../models/person';
import { Action, Dispatch } from 'redux';
import { FetchError, FetchSuccess, RestActions, STATUS } from './utils';
import { getPerson } from '../api/person-api';
import { aremark } from '../mock/person-mock';

export interface PersonState {
    data: Person;
    status: STATUS;
    error: string | null;
}

const reducerName = 'person/';

const actionNames = {
    PENDING: reducerName + STATUS.PENDING,
    OK: reducerName + STATUS.OK,
    ERROR: reducerName + STATUS.ERROR
};

const initialState: PersonState = {
    data: aremark,
    status: STATUS.NOT_STARTED,
    error: null
};

export function hentPerson(fodselsnummer: string) {
    return (dispatch: Dispatch<Action>) => {
        dispatch({ type: actionNames.PENDING});
        return getPerson(fodselsnummer)
            .then((person) => {
                dispatch({type: actionNames.OK, data: person});
                return person;
            })
            .catch((err) => {
                dispatch({type: actionNames.ERROR, error: err});
            });
    };
}

export default function reducer(state: PersonState = initialState, action: RestActions<Person>): PersonState {
    switch (action.type) {
        case actionNames.PENDING:
            return {...state, status: STATUS.PENDING };
        case actionNames.OK:
            return {...state, status: STATUS.OK, data: (<FetchSuccess<Person>> action).data};
        case actionNames.ERROR:
            return {...state, status: STATUS.ERROR, error: (<FetchError> action).error};
        default:
            return state;
    }
}