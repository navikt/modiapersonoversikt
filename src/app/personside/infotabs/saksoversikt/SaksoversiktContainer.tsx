import * as React from 'react';
import { isNotStarted, RestResource } from '../../../../redux/restReducers/restResource';
import { Sakstema, SakstemaResponse } from '../../../../models/saksoversikt/sakstema';
import styled from 'styled-components';
import theme from '../../../../styles/personOversiktTheme';
import { AppState } from '../../../../redux/reducers';
import { connect } from 'react-redux';
import { hentSaksoversikt, reloadSaksoversikt } from '../../../../redux/restReducers/saksoversikt';
import Innholdslaster from '../../../../components/Innholdslaster';
import { BaseUrlsResponse } from '../../../../models/baseurls';
import { hentBaseUrls } from '../../../../redux/restReducers/baseurls';
import { AsyncDispatch } from '../../../../redux/ThunkTypes';
import { PersonRespons } from '../../../../models/person/person';
import DokumentOgVedlegg from './dokumentvisning/DokumentOgVedlegg';
import SakstemaListeContainer from './sakstemaliste/SakstemaListeContainer';
import SaksDokumenterContainer from './saksdokumenter/SaksDokumenterContainer';
import { settVisDokument } from '../../../../redux/saksoversikt/actions';
import VisuallyHiddenAutoFokusHeader from '../../../../components/VisuallyHiddenAutoFokusHeader';

interface StateProps {
    baseUrlResource: RestResource<BaseUrlsResponse>;
    saksoversiktResource: RestResource<SakstemaResponse>;
    personResource: RestResource<PersonRespons>;
    visDokument: boolean;
    valgtSakstema?: Sakstema;
}

interface DispatchProps {
    hentBaseUrls: () => void;
    hentSaksoversikt: (fødselsnummer: string) => void;
    reloadSaksoversikt: (fødselsnummer: string) => void;
    skjulDokumentOgVisSaksoversikt: () => void;
}

interface OwnProps {
    fødselsnummer: string;
}

type Props = StateProps & DispatchProps & OwnProps;

export const saksoversiktMediaTreshold = '80rem';

const SaksoversiktArticle = styled.article`
    @media (min-width: ${saksoversiktMediaTreshold}) {
        display: flex;
        align-items: flex-start;
        > *:last-child {
            margin-left: ${theme.margin.layout};
        }
    }
    .visually-hidden {
        ${theme.visuallyHidden}
    }
    > * {
        margin-bottom: ${theme.margin.layout};
    }
`;

class SaksoversiktContainer extends React.PureComponent<Props> {
    componentDidMount() {
        this.props.skjulDokumentOgVisSaksoversikt();
        if (isNotStarted(this.props.baseUrlResource)) {
            this.props.hentBaseUrls();
        }
        if (isNotStarted(this.props.saksoversiktResource)) {
            this.props.hentSaksoversikt(this.props.fødselsnummer);
        }
    }

    render() {
        if (this.props.visDokument) {
            return <DokumentOgVedlegg />;
        } else {
            return (
                <SaksoversiktArticle aria-label="Brukerens saker">
                    <VisuallyHiddenAutoFokusHeader tittel="Brukerens saker" />
                    <Innholdslaster avhengigheter={[this.props.saksoversiktResource, this.props.baseUrlResource]}>
                        <SakstemaListeContainer />
                        <SaksDokumenterContainer />
                    </Innholdslaster>
                </SaksoversiktArticle>
            );
        }
    }
}

function mapStateToProps(state: AppState): StateProps {
    return {
        baseUrlResource: state.restResources.baseUrl,
        saksoversiktResource: state.restResources.sakstema,
        personResource: state.restResources.personinformasjon,
        visDokument: state.saksoversikt.visDokument,
        valgtSakstema: state.saksoversikt.valgtSakstema
    };
}

function mapDispatchToProps(dispatch: AsyncDispatch): DispatchProps {
    return {
        hentBaseUrls: () => dispatch(hentBaseUrls()),
        hentSaksoversikt: (fødselsnummer: string) => dispatch(hentSaksoversikt(fødselsnummer)),
        reloadSaksoversikt: (fødselsnummer: string) => dispatch(reloadSaksoversikt(fødselsnummer)),
        skjulDokumentOgVisSaksoversikt: () => dispatch(settVisDokument(false))
    };
}

export default connect(
    mapStateToProps,
    mapDispatchToProps
)(SaksoversiktContainer);
