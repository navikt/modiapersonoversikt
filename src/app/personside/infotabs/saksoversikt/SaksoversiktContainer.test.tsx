import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TestProvider from '../../../../test/Testprovider';
import { aremark } from '../../../../mock/person/aremark';
import SaksoversiktContainer from './SaksoversiktContainer';

test('Viser saksoversiktcontainer med alt innhold', () => {
    const container = renderer.create(
        <TestProvider>
            <SaksoversiktContainer fødselsnummer={aremark.fødselsnummer} />
        </TestProvider>
    );

    const json = container.toJSON();
    expect(json).toMatchSnapshot();
});
