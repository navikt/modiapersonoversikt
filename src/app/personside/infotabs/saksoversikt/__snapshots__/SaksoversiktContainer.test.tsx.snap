// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viser saksoversiktcontainer med alt innhold 1`] = `
.c15 {
  text-align: center;
}

.c17 {
  text-transform: uppercase;
}

.c16 {
  font-weight: bold;
}

.c22 {
  position: relative;
  border: none;
  padding: 0;
  border-radius: .35rem;
  cursor: pointer;
  background-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #0067c5;
}

.c22:after {
  border-bottom: 1px #b7b1a9 solid;
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%;
}

.c22:focus {
  outline: none;
  box-shadow: 0 0 0 0.1875rem #FFBD66;
}

.c22:hover {
  opacity: 0.8;
}

.c22:hover:after {
  content: '';
  border-color: #0067c5;
}

.c8 {
  padding: 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
}

.c8:hover {
  box-shadow: inset 0 0 0 0.1rem #0067c5,0 0.1rem 0.1rem rgba(0,0,0,0.2);
}

.c8:hover > button:last-child {
  padding-left: 0.5rem;
}

.c8 > *:first-child {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c4 {
  padding: 1.25rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  background-color: #cce1f3;
}

.c4:hover {
  box-shadow: inset 0 0 0 0.1rem #0067c5,0 0.1rem 0.1rem rgba(0,0,0,0.2);
}

.c4:hover > button:last-child {
  padding-left: 0.5rem;
}

.c4 > *:first-child {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c7 {
  border: none;
  padding: 0;
  height: 2rem;
  width: 2rem;
  border-radius: .35rem;
  cursor: pointer;
  background-color: transparent;
  -webkit-transition: padding-left 0.3s;
  transition: padding-left 0.3s;
}

.c7:focus {
  outline: none;
  box-shadow: 0 0 0 0.1875rem #FFBD66;
}

.c6 svg {
  height: 1.875rem;
  width: 1.875rem;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c5 .order-first {
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
}

.c5 .order-second {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c3 > * {
  border-top: solid 0.0625rem #b7b1a9;
}

.c1 {
  border-radius: 0.25rem;
  background-color: white;
  box-shadow: 0 0 0.5rem rgba(0,0,0,0.1);
  min-width: 24rem;
  -webkit-flex-basis: 24rem;
  -ms-flex-preferred-size: 24rem;
  flex-basis: 24rem;
}

.c1 ol {
  list-style: none;
}

.c2 {
  padding: 1.25rem;
}

.c23 {
  color: #78706a;
  margin-top: 3px;
}

.c23 .typo-etikett-liten {
  line-height: 1rem;
}

.c18 {
  padding: 1.25rem 0.625rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c20 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c20 > *:not(:last-child) {
  margin-bottom: 0.5rem;
}

.c19 {
  padding-right: 1rem;
}

.c19 svg {
  height: 1.875rem;
  width: 1.875rem;
}

.c21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c21 .order-first {
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
}

.c21 .order-second {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c9 {
  position: relative;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c10 {
  border-radius: 0.25rem;
  background-color: white;
  box-shadow: 0 0 0.5rem rgba(0,0,0,0.1);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 1.25rem;
}

.c10 > *:first-child {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c10 > div {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c10 > div > * {
  margin-bottom: 0.5rem;
}

.c10 > div:last-child {
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
}

.c13 {
  padding: 0;
  margin: 0;
  list-style: none;
}

.c14 > *:first-child {
  padding: 0.2rem 0.625rem;
}

.c14 ol {
  border-radius: 0.25rem;
  background-color: white;
  box-shadow: 0 0 0.5rem rgba(0,0,0,0.1);
  padding: 0;
  margin: 0;
}

.c14 ol > *:not(:first-child) {
  border-top: solid 0.0625rem #b7b1a9;
}

.c12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c12 > *:not(:last-child) {
  padding-right: 1rem;
}

.c12 > * {
  margin-bottom: 0;
}

.c24 {
  margin-top: 2rem;
}

.c11 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c11 > *:last-child {
  margin-left: 1rem;
}

.c11:focus {
  outline: none;
}

.c0 .visually-hidden {
  position: absolute !important;
  height: 1px;
  width: 1px;
  overflow: hidden;
  -webkit-clip: rect(1px 1px 1px 1px);
  clip: rect(1px 1px 1px 1px);
  -webkit-clip: rect(1px,1px,1px,1px);
  clip: rect(1px,1px,1px,1px);
}

.c0 > * {
  margin-bottom: .8rem;
}

@media (min-width:80rem) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: flex-start;
    -webkit-box-align: flex-start;
    -ms-flex-align: flex-start;
    align-items: flex-start;
  }

  .c0 > *:last-child {
    margin-left: .8rem;
  }
}

<article
  aria-label="Brukerens saker"
  className="c0"
>
  <h2
    className="visually-hidden"
    tabIndex={-1}
  >
    Brukerens saker
  </h2>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <h2
        className="typo-undertittel"
      >
        Tema
      </h2>
    </div>
    <nav
      aria-label="Velg sakstema"
    >
      <ol
        className="c3"
      >
        <li>
          <div
            className="c4"
            onClick={[Function]}
          >
            <div>
              <div
                className="c5"
              >
                <p
                  className="typo-element order-second"
                >
                  Alle tema
                </p>
                <p
                  className="typo-normal order-first"
                >
                  08.02.2018
                </p>
              </div>
            </div>
            <span
              className="c6"
            />
            <button
              aria-label="Vis Alle tema"
              aria-selected={true}
              className="c7"
              onClick={[Function]}
            >
              <i
                className="nav-frontend-chevron chevronboks chevron--stor chevron--hoyre"
              />
            </button>
          </div>
        </li>
        <li>
          <div
            className="c8"
            onClick={[Function]}
          >
            <div>
              <div
                className="c5"
              >
                <p
                  className="typo-element order-second"
                >
                  Sykepenger
                </p>
                <p
                  className="typo-normal order-first"
                >
                  08.02.2018
                </p>
              </div>
              <p
                className="typo-normal"
              >
                1
                 
                søknad
                 er 
                ferdig behandlet
                .
              </p>
            </div>
            <span
              className="c6"
            >
              <svg
                contentScriptType="text/ecmascript"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#C6C2BF"
                >
                  <path
                    d="M17.5 10c1.3 0 2.5.3 3.5.9V4.5c0-.3-.2-.5-.5-.5H19V2.5c0-.3-.2-.5-.5-.5H6V.5c0-.3-.2-.5-.5-.5h-5C.2 0 0 .2 0 .5V16c0 1.1.9 2 2 2h8v-.5c0-4.1 3.4-7.5 7.5-7.5zM3 4.5V16a1 1 0 0 1-2 0V1h4v1.5c0 .3.2.5.5.5H18v1H3.5c-.3 0-.5.2-.5.5z"
                  />
                  <path
                    d="M17.5 11a6.5 6.5 0 0 0-5 10.7l9.2-9.1a6.5 6.5 0 0 0-4.2-1.6zM22.4 13.3l-9.1 9.1c1.1 1 2.6 1.6 4.2 1.6a6.5 6.5 0 0 0 5-10.7z"
                  />
                </g>
              </svg>
            </span>
            <button
              aria-label="Vis Sykepenger"
              aria-selected={false}
              className="c7"
              onClick={[Function]}
            >
              <i
                className="nav-frontend-chevron chevronboks chevron--stor chevron--hoyre"
              />
            </button>
          </div>
        </li>
        <li>
          <div
            className="c8"
            onClick={[Function]}
          >
            <div>
              <div
                className="c5"
              >
                <p
                  className="typo-element order-second"
                >
                  Tilleggsstønad arbeidssøkere
                </p>
                <p
                  className="typo-normal order-first"
                >
                  21.11.2017
                </p>
              </div>
            </div>
            <span
              className="c6"
            >
              <svg
                contentScriptType="text/ecmascript"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#C6C2BF"
                >
                  <path
                    d="M17.5 10c1.3 0 2.5.3 3.5.9V4.5c0-.3-.2-.5-.5-.5H19V2.5c0-.3-.2-.5-.5-.5H6V.5c0-.3-.2-.5-.5-.5h-5C.2 0 0 .2 0 .5V16c0 1.1.9 2 2 2h8v-.5c0-4.1 3.4-7.5 7.5-7.5zM3 4.5V16a1 1 0 0 1-2 0V1h4v1.5c0 .3.2.5.5.5H18v1H3.5c-.3 0-.5.2-.5.5z"
                  />
                  <path
                    d="M17.5 11a6.5 6.5 0 0 0-5 10.7l9.2-9.1a6.5 6.5 0 0 0-4.2-1.6zM22.4 13.3l-9.1 9.1c1.1 1 2.6 1.6 4.2 1.6a6.5 6.5 0 0 0 5-10.7z"
                  />
                </g>
              </svg>
            </span>
            <button
              aria-label="Vis Tilleggsstønad arbeidssøkere"
              aria-selected={false}
              className="c7"
              onClick={[Function]}
            >
              <i
                className="nav-frontend-chevron chevronboks chevron--stor chevron--hoyre"
              />
            </button>
          </div>
        </li>
      </ol>
    </nav>
  </div>
  <section
    aria-label="Saksdokumenter for Alle tema"
    className="c9"
  >
    <section
      className="c10"
    >
      <div>
        <div
          className="c11"
          tabIndex={-1}
        >
          <h2
            className="typo-undertittel"
          >
            Alle tema
          </h2>
          <p
            className="typo-normal"
          >
            (
            2
             journalposter)
          </p>
        </div>
        <form
          aria-label="Filter"
          className="c12"
        >
          <div
            className="skjemaelement skjemaelement--horisontal"
          >
            <input
              checked={true}
              className="skjemaelement__input checkboks"
              id="Helt tilfeldig ID"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              className="skjemaelement__label"
              htmlFor="Helt tilfeldig ID"
            >
              Bruker
            </label>
            <div
              aria-live="assertive"
              role="alert"
            />
          </div>
          <div
            className="skjemaelement skjemaelement--horisontal"
          >
            <input
              checked={true}
              className="skjemaelement__input checkboks"
              id="Helt tilfeldig ID"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              className="skjemaelement__label"
              htmlFor="Helt tilfeldig ID"
            >
              NAV
            </label>
            <div
              aria-live="assertive"
              role="alert"
            />
          </div>
          <div
            className="skjemaelement skjemaelement--horisontal"
          >
            <input
              checked={true}
              className="skjemaelement__input checkboks"
              id="Helt tilfeldig ID"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              className="skjemaelement__label"
              htmlFor="Helt tilfeldig ID"
            >
              Andre
            </label>
            <div
              aria-live="assertive"
              role="alert"
            />
          </div>
        </form>
      </div>
      <div>
        <a
          className="lenke"
          href="https://norg2-frontend.nais.preprod.local/#/startsok?tema=SYK&gt=0118"
          rel="noopener noreferrer"
          target="_blank"
        >
          <p
            className="typo-normal"
          >
            Oversikt over enheter og tema de behandler
          </p>
        </a>
      </div>
    </section>
    <ol
      aria-label="Dokumenter gruppert på årstall"
      className="c13"
    >
      <li
        className="c14"
      >
        <h3
          className="typo-normal"
        >
          <div
            className="c15"
          >
            <span
              className="c16"
            >
              <span
                className="c17"
              >
                2018
              </span>
            </span>
          </div>
        </h3>
        <ol>
          <li
            className="c18"
            onClick={[Function]}
          >
            <div
              className="c19"
            >
              <svg
                aria-label="Du har ikke tilgang til dette dokumentet"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#C6C2BF"
                >
                  <path
                    d="M17 10V5.5l-.1-.4-5-5-.4-.1H.5C.2 0 0 .2 0 .5v21c0 .3.2.5.5.5h11A7.5 7.5 0 0 1 17 10zM11.5.5l5 5h-5v-5z"
                  />
                  <path
                    d="M17.5 11a6.5 6.5 0 0 0-5 10.7l9.2-9.1a6.5 6.5 0 0 0-4.2-1.6zM22.4 13.3l-9.1 9.1c1.1 1 2.6 1.6 4.2 1.6a6.5 6.5 0 0 0 5-10.7z"
                  />
                </g>
              </svg>
            </div>
            <div
              className="c20"
            >
              <div
                className="c21"
              >
                <div
                  className="order-second"
                >
                  <button
                    aria-disabled={true}
                    className="c22"
                    onClick={[Function]}
                  >
                    <p
                      className="typo-element"
                    >
                      Vurdering feilutbetaling/revurdering
                    </p>
                  </button>
                </div>
                <div
                  className="order-first"
                >
                  <p
                    className="typo-normal"
                  >
                    08.02.2018 / Fra Marie
                  </p>
                </div>
              </div>
              
              <div
                className="c23"
              >
                <p
                  className="typo-etikett-liten"
                >
                  Sykepenger
                   / Saksid: 
                  kv3uqgr5
                </p>
              </div>
            </div>
          </li>
        </ol>
      </li>
      <li
        className="c14"
      >
        <h3
          className="typo-normal"
        >
          <div
            className="c15"
          >
            <span
              className="c16"
            >
              <span
                className="c17"
              >
                2017
              </span>
            </span>
          </div>
        </h3>
        <ol>
          <li
            className="c18"
            onClick={[Function]}
          >
            <div
              className="c19"
            >
              <svg
                aria-label="Du har ikke tilgang til dette dokumentet"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#C6C2BF"
                >
                  <path
                    d="M17 10V5.5l-.1-.4-5-5-.4-.1H.5C.2 0 0 .2 0 .5v21c0 .3.2.5.5.5h11A7.5 7.5 0 0 1 17 10zM11.5.5l5 5h-5v-5z"
                  />
                  <path
                    d="M17.5 11a6.5 6.5 0 0 0-5 10.7l9.2-9.1a6.5 6.5 0 0 0-4.2-1.6zM22.4 13.3l-9.1 9.1c1.1 1 2.6 1.6 4.2 1.6a6.5 6.5 0 0 0 5-10.7z"
                  />
                </g>
              </svg>
            </div>
            <div
              className="c20"
            >
              <div
                className="c21"
              >
                <div
                  className="order-second"
                >
                  <button
                    aria-disabled={true}
                    className="c22"
                    onClick={[Function]}
                  >
                    <p
                      className="typo-element"
                    >
                      Spørsmål via nav.no
                    </p>
                  </button>
                </div>
                <div
                  className="order-first"
                >
                  <p
                    className="typo-normal"
                  >
                    21.11.2017 / Samtalereferat
                  </p>
                </div>
              </div>
              
              <div
                className="c23"
              >
                <p
                  className="typo-etikett-liten"
                >
                  Sykepenger
                   / Saksid: 
                  8qj6vuv6
                </p>
              </div>
            </div>
          </li>
        </ol>
      </li>
    </ol>
    <div
      className="c24"
    />
    <div
      className="alertstripe alertstripe--info"
    >
      <span
        aria-label="info"
        className="alertstripe__ikon"
      >
        <svg
          focusable="false"
          height="1.5em"
          kind="info-sirkel-fyll"
          viewBox="0 0 24 24"
          width="1.5em"
        >
          <g
            fill="none"
            fillRule="evenodd"
          >
            <path
              d="M12.205-.004l-.214.002a12.225 12.225 0 0 0-8.517 3.659C1.179 5.977-.053 9.013.002 12.208c.115 6.613 5.296 11.793 11.795 11.793l.212-.002c6.726-.116 12.105-5.595 11.99-12.21C23.883 5.178 18.702-.003 12.204-.003z"
              fill="#FFA733"
              fillRule="nonzero"
            />
            <path
              d="M12.027 19H12A1.499 1.499 0 0 1 11.973 16L12 16a1.501 1.501 0 0 1 .027 3z"
              fill="#3E3832"
            />
            <path
              d="M12 5a1 1 0 0 1 1 1v7a1 1 0 0 1-2 0V6a1 1 0 0 1 1-1z"
              fill="#3E3832"
              fillRule="nonzero"
            />
          </g>
        </svg>
      </span>
      <span
        className="typo-normal alertstripe__tekst"
      >
        Modia viser elektroniske dokumenter brukeren har sendt inn via nav.no etter 9. desember 2014. Dokumenter som er journalført vises fra og med 4.juni 2016
      </span>
    </div>
  </section>
</article>
`;
